templates/jupyter.sh.in

```diff
@@ -21,7 +21,7 @@ echo '
     "display_name": "Simons Observatory @VERSION@",
     "env": {
         "PATH": "@ENVPREFIX@/bin",
-        "PYTHONPATH": "@ENVPREFIX@/lib/python@PYVER@/site-packages"
+        "PYTHONPATH": "${HOME}/.local/soconda/@VERSION@/lib/python@PYVER@/site-packages:@ENVPREFIX@/lib/python@PYVER@/site-packages"
```

Comment:
May I ask why adding `${HOME}/.local/soconda/@VERSION@/lib/python@PYVER@/site-packages` to search path?

Reply:
In our module files, we set PYTHONUSERBASE to this versioned location.  Many users do a `pip install --user sotodlib` or install other packages that they want to have at the front of the search path.  This change ensures that they can use those packages in the jupyter session.  I tested that this works, although I did not test if setting PYTHONUSERBASE in the kernel spec has the same effect.  I could try that approach instead if you prefer

Reply:
Thanks for clarification. I think set PYTHONUSERBASE may be better, it will run any `*.pth` files[1] if installed by any packages, this won't be the case for PYTHONPATH[2].

If we want to use PYTHONUSERBASE, we need to __remove__ the `"PYTHONPATH":` line. Because PYTHONPATH will be inserted at the front of search path. I think this is fine, because we already specified python at [here](https://github.com/simonsobs/soconda/blob/1597340ff8db283fb6f3e30b51d05734ada16ed8/templates/jupyter.sh.in#L15). And I did some quick test without PYTHONPATH in jupyter kernel, it would add `@ENVPREFIX@/lib/python@PYVER@/site-packages` to [`sys.path`](https://docs.python.org/3/library/sys.html#sys.path).


[1] https://docs.python.org/3/library/site.html#site.USER_SITE
[2] https://peps.python.org/pep-0370/

Reply:
Ok, this should be fixed now.